add-automatic-updating

jujunya opened the pull request:
#what
chat-spaceにおける、自動更新機能の実装

#why
chat-space実装の為

Commit:
add-automatic-updating

#what
chat-spaceにおける、自動更新機能の実装

#why
chat-space実装の為

hzak2xx commented on app/assets/javascripts/message.js:
```diff
@@ -37,13 +39,42 @@ $(function(){
+        .done(function(data) {
+          var insertHTML = '';
+          data.forEach(function(message) {
+            if ((message.id > lastMessageId) || (lastMessageId == undefined) ){
```
フロントでid比較処理するのではなく、ajaxでidを送っているのでそちらでしましょう

jujunya replied:
修正しました。確認宜しくお願い致します。

jujunya replied:
余計な修正部分も少し入ってしまい、若干commit内容が見にくくなってしまいました。
申し訳ありません。

Commit:
【修正】非同期自動更新機能

#what
レビューにて指摘のあった、id比較処理の場所について修正
massage.js → message#index アクション

#why
chat-space自動更新機能実装のため

miyagit commented on app/views/messages/_message.html.haml:
```diff
@@ -1,11 +1,12 @@
+      .upper-message__user-name
+        = message.user.name
+      .upper-message_date
+        = message.created_at.strftime("%Y/%m/%d/ %H:%M")
```
railsのtime_formatというものがあるので、それをつかいましょう。

jujunya replied:
修正しました。確認宜しくお願い致します。

Commit:
【修正2】非同期自動更新機能

#what
railsのtime_format を利用

#why
chat-space自動更新機能実装の為
